app/constants/offer-sorting-options.js

```diff
@@ -0,0 +1,30 @@
+export const offerSortingOptions = [
+  {
+    column_name: "schedules.scheduled_at",
+    column_alias: "Due date",
```

Comment:
We probably want to have that translated, you can maybe input the translation key here